kreyol/kreyol.md

```diff
@@ -0,0 +1,10 @@
+### Neg Tout moun ta renmen genyen an ###
+
+Sa'a ta sipoze listwa on neg , on gwo jen gason , on kokenchen neg brenn. 
+Non neg saa se #### Tousen Louveti #### . An franse ou te ka di : #### Toussaint Louverture. ####
+Yon neg ki te telman entelijan , li rive enfiltre tout ti klan enemi ayisyen, neg sa a rive goumen , pou langlete, ki te on pi gwo pisans nan epok sa, 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.